projects/common/src/telemetry/telemetry.ts

```diff
@@ -20,6 +20,7 @@ export interface UserTelemetryProvider<TInitConfig = unknown> {
 
 export interface TelemetryProviderConfig {
   orgId: string;
+  orgTrackingURL?: string;
```

Comment:
As discussed in the sync, please add this in RudderTelemetryInitConfig which would extend from TelemetryProviderConfig